File: Bubble_Sort.py
def bubblesort(elements):
    swapped = False
    for n in range(len(elements)-1, 0, -1):
        for i in range(n):
            if elements[i] > elements[i + 1]:
                swapped = True
                elements[i], elements[i + 1] = elements[i + 1], elements[i]       
        if not swapped:
            return
 
elements = eval(input("Enter the elements of array within '[]'" : ))
print("Unsorted list is,")
print(elements)
bubblesort(elements)
print("Sorted Array is, ")
print(elements)

File: Java/BubbleSort.java
import java.util.Scanner;
public class BubbleSort
{
   public static void main(String[] args)
   {
      int i, j, x;
      Scanner s = new Scanner(System.in);
      System.out.print("Enter the size of the array : ");
      int n = s.nextInt();
      int[] array = new int[n];
      System.out.print("Enter the Elements : ");
      for(i=0; i<n; i++)
      {
         array[i] = s.nextInt();
      }
      for(i=0; i<(n-1); i++)
      {
         for(j=0; j<(n-i-1); j++)
         {
            if(array[j]>array[j+1])
            {
               x = array[j];
               array[j] = array[j+1];
               array[j+1] = x;
            }
         }
      }
      System.out.println("The sorted array is:");
      for(i=0; i<n; i++)
         System.out.print(array[i]+ " ");
   }
}